src/privatesend/privatesend-client.h

```diff
@@ -224,8 +226,10 @@ class CPrivateSendClientManager
     bool CheckAutomaticBackup();
 
 public:
-    int nCachedNumBlocks;    //used for the overview screen
-    bool fCreateAutoBackups; //builtin support for automatic backups
+    // used for the overview screen
```

Comment:
Those "move comments around" changes should probably be reverted since they are unrelated to this PR?

Reply:
I mean... They were related until Udjin refactored nSalt out of this class

Reply:
resolved by latest commit